scripts/version.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+FLUTTER_REQUIRED_VERSION=$(grep -o '"flutterSdkVersion": "[^"]*' .fvm/fvm_config.json | awk -F': "' '{print $2}')
+
+fvm use $FLUTTER_REQUIRED_VERSION
```

Comment:
Since we are adding `fvm` to the repository, this is no longer necessary, we can simply call `fvm use`

Reply:
\+